driver/pace/driver/diagnostics.py

```diff
@@ -102,9 +110,26 @@ def store(self, time: Union[datetime, timedelta], state: DriverState):
                 except AttributeError:
                     quantity = getattr(state.physics_state, name)
                 zarr_state[name] = quantity
+            derived_state = self._get_derived_state(state)
+            zarr_state.update(derived_state)
             assert time is not None
             self.monitor.store(zarr_state)
 
+    def _get_derived_state(self, state: DriverState):
+        output = {}
+        if len(self.derived_names) > 0:
+            for name in self.derived_names:
+                if "column_integrated_" in name:
+                    tracer = name.split("column_integrated_")[-1]
```

Comment:
```suggestion
                    tracer = tracer[len("column_integrated_"):]
```

Reply:
I think it should be `name[len("column_integrated_"):]`